ydb/mvp/oidc_proxy/oidc_session_create_nebius.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "oidc_session_create.h"
+
+namespace NMVP {
+
+class THandlerSessionCreateNebius : public THandlerSessionCreate {
+private:
+    using TBase = THandlerSessionCreate;
+
+public:
+    THandlerSessionCreateNebius(const NActors::TActorId& sender,
+                                const NHttp::THttpIncomingRequestPtr& request,
+                                const NActors::TActorId& httpProxyId,
+                                const TOpenIdConnectSettings& settings)
+        : THandlerSessionCreate(sender, request, httpProxyId, settings)
+        {}
+
+    void RemoveAppliedCookie(const TString& cookieName) override {
+        ResponseHeaders.Set("Set-Cookie", TStringBuilder() << cookieName << "=; Path=" << GetAuthCallbackUrl() << "; Max-Age=0");
+        ResponseHeaders.Set("Set-Cookie", TStringBuilder() << CreateNameSessionCookie(Settings.ClientId) << "=; Max-Age=0");
+    }
+
+    void RequestSessionToken(const TString& code, const NActors::TActorContext& ctx) override {
+        TStringBuilder body;
+        TStringBuf host = Request->Host;
+        body << "code=" << code
+                << "&client_id=" << Settings.ClientId
+                << "&grant_type=authorization_code"
+                << "&redirect_uri="
+                << (Request->Endpoint->Secure ? "https://" : "http://")
```

Comment:
token endpoint url is our setting? We must always go there with the same protocol independently from client request

Reply:
it's no token endpoint. in redirect_uri must be link to `oidc_proxy/auth/callback`